plugin-gestures/src/main/java/com/mapbox/maps/plugin/gestures/GesturesPluginImpl.kt

```diff
@@ -1042,7 +1041,7 @@ class GesturesPluginImpl : GesturesPlugin, GesturesSettingsBase, MapStyleObserve
     cancelTransitionsIfRequired()
 
     // disabling move gesture during shove
-    gesturesManager.moveGestureDetector.isEnabled = false
+    gestureState.saveAndDisable(GestureState.Type.Shove)
```

Comment:
From the comment we are disabling move gesture, but from the code it looks like we are disabling shove..
Shall we rename the function e.g. `gestureState.saveAndDisable(GestureState.Type.Move)`?

Reply:
Comment only was confusing, will remove old comments. Code is correct, since we are passing `Type.Shove` to `saveAndDisable` and there we disable `moveGestureDetector`, same as in the original code.